packages/app/src/App.tsx

```diff
@@ -20,6 +20,7 @@ import {
   OAuthRequestDialog,
   SignInPage,
 } from '@backstage/core-components';
+import { HomeIndexPage } from '@backstage/plugin-home';
```

Comment:
Super nit picky, but having a plugin exportable component called `HomeIndexPage` and then a local exported component in my app called `HomePage` does seem like over time (and at scale with a large number of Backstage integrators) it may potentially become confusing. I almost wonder if the plugin should export a name a bit further differentiated, something like `ComposableHomePage` or something?

i.e.,

```typescript
import { ComposableHomePage } from '@backstage/plugin-home';
// ...
import { HomePage } from './components/home/HomePage';
// ...
<Route path="/" element={<ComposableHomePage />}>
       <HomePage />
     </Route>
```

Reply:
In some places we've started naming the out-of-the-box default impl of something `Default*` which has the benefit of (hopefully) clearly signaling its explicit replaceability. Tossing `DefaultHomepageRoot` or `DefaultHomepageCompositionRoot`  into the suggestions box